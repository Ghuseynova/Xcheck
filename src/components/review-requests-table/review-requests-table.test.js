import React from 'react';
import { shallow } from 'enzyme';
import ReviewRequestsTableCreation from './review-requests-table';

describe('render ', () => {
  test('Review requests table creation', () => {
    const tree = shallow(<ReviewRequestsTableCreation/>);
	  expect(tree).toMatchSnapshot();
	});
});