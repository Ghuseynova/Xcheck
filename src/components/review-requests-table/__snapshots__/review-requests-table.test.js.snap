// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render  Review requests table creation 1`] = `ShallowWrapper {}`;
